tiledb/bioimg/converters/base.py

```diff
@@ -205,51 +206,50 @@ def from_tiledb(
 
-        with tiledb.scope_ctx(config):
-            out_uri_res, scheme = resolve_path(output_path)
-            vfs_use = False if is_local_path(scheme) else True
+        out_uri_res, scheme = resolve_path(output_path)
+        vfs_use = False if is_local_path(scheme) else True
 
-            # OS specific
-            destination_uri = (
-                os.path.join(scratch_space, os.path.basename(out_uri_res))
-                if vfs_use
-                else out_uri_res
-            )
-            slide = TileDBOpenSlide(input_path, attr=attr)
-            writer = cls._ImageWriterType(destination_uri)
+        # OS specific
+        destination_uri = (
+            os.path.join(scratch_space, os.path.basename(out_uri_res))
+            if vfs_use
+            else out_uri_res
+        )
 
-            with slide, writer:
-                writer.write_group_metadata(slide.properties)
-                group_metadata = jsonpickle.loads(
-                    slide.properties.get("metadata", "{}")
+        slide = TileDBOpenSlide(input_path, attr=attr, config=config)
+        writer = cls._ImageWriterType(destination_uri)
+
+        with slide, writer:
+            writer.write_group_metadata(slide.properties)
+            group_metadata = jsonpickle.loads(slide.properties.get("metadata", "{}"))
+            for idx, _ in enumerate(slide.levels):
+                if idx < level_min:
+                    continue
+                level_image = slide.read_level(idx, to_original_axes=True)
+                level_metadata = writer.compute_level_metadata(
+                    idx == level_min,
+                    len(slide.levels) - level_min,
+                    level_image.dtype,
+                    group_metadata,
+                    slide.level_properties(idx),
+                    **writer_kwargs,
+                )
+                writer.write_level_image(
+                    level_image,
+                    level_metadata,
                 )
-                for idx, _ in enumerate(slide.levels):
-                    if idx < level_min:
-                        continue
-                    level_image = slide.read_level(idx, to_original_axes=True)
-                    level_metadata = writer.compute_level_metadata(
-                        idx == level_min,
-                        len(slide.levels) - level_min,
-                        level_image.dtype,
-                        group_metadata,
-                        slide.level_properties(idx),
-                        **writer_kwargs,
-                    )
-                    writer.write_level_image(
-                        level_image,
-                        level_metadata,
-                    )
 
             if vfs_use:
                 # Flush to remote
                 with open(destination_uri, "rb") as data:
-                    vfs = tiledb.VFS()
+                    vfs = tiledb.VFS(config=output_config)
```

Comment:
I think we need some condition to determine to use `output_config` or just `config`. If the user only pass `config` (deprecated but supported) won't we ignore it now?

Reply:
I see the following scenarios:
1. **The user uses an older version, where we do not support separate configuration for the exported path.** (👍 )
2. **The user uses the updated version, where he wants to export from remote source to local output path**, thus uses only `config` arg. The `vfs_use` is becoming `False` as the output path is local and the `destination_uri` gets the local value from here 
```python
destination_uri = (
            os.path.join(scratch_space, os.path.basename(out_uri_res))
            if vfs_use
            else out_uri_res
        )
```
and the writer following writes directly to this local output path. ( 👍 )
3. **The user wants to export to a remote and the source is also remote.** Then normally he should use one `config` for source and one `output_config` for the destination  if the configs are different, while also providing a remote `output_path`. The read will happen and the intermediate file will be stored in the `scratch space`, and now what happens next is upon us. In this implementation if the `output_config` is not given and as said the `output_path` is also remote we fail in the `vfs.open` where the `config=output_config=None` we will not be able to access the remote destination path due to missing credentials. Now the question in semantics is if we want to silently assign `config` to `output_config` if the latter is not defined as a default behaviour, thinking that the source and dest are under the same access credentials policy. (👍 )
4. For the local cases for remote and dest should work fine with `configs=None`(👍 )

Is there any case which I missed?

Reply:
So given that with current version user does not have the ability to use different configs, let's follow the semantics I described earlier concerning the following.

>  Now the question in semantics is if we want to silently assign config to output_config if the latter is not defined as a default behaviour, thinking that the source and dest are under the same access credentials policy. (👍 )

This way the user that upgrades and uses only config without knowing the existence of the new features will be able to export to a remote source, but he/she is limited by the package version, where both the source and destination should comply to the same credential access. Would it be sufficient for compatibility?